catalogue/webapp/app.js

```diff
@@ -25,6 +25,7 @@ module.exports = app
     route('/works/progress', '/progress', router, app);
     route('/works/:id', '/work', router, app);
     route('/works', '/works', router, app);
+    route('/images', '/images', router, app);
```

Comment:
Could we please add
```
<meta name="robots" content="noindex">
```

To the page somewhere?

Reply:
I imagine we might have something along the lines of a search component.
`/works` => `<SearchComponent search={'images' | 'works'} />`

Which makes me think we might want to start thinking about a `/search` endpoint.

Where this starts to make sense is the search bar / the way filters etc work should probably ubiquitous across the experience, and specific search mechanism and search results should be specific.